utils/cosmos/iscn/sign.ts

```diff
@@ -169,7 +169,7 @@ export async function signISCN(
     ? signingClient.updateISCNRecord(address, iscnId as string, tx, {
         memo: memo || 'app.like.co',
         fee: {
-          gas,
+          gas: new BigNumber(gas).multipliedBy(ISCN_GAS_MULTIPLIER).toFixed(0),
```

Comment:
Just increase the default `ISCN_GAS_FEE` by `ISCN_GAS_MULTIPLIER`? `gas` as a input is expected to be override, `ISCN_GAS_MULTIPLIER` should be applied to the input not inside this function